### ISSUE TITLE
Ugly modeline at startup

### ISSUE THREAD

**Issue by mphojele** [syl20bnr/spacemacs#13154]
#### Description :octocat:
The modeline looks good only after I've either opened a new buffer or ran some helm commands. I helm commands because a mere `M-:` doesn't fix the modeline. I don't have this issue when I run a emacs daemon and summon a client.

#### Reproduction guide :beetle:
- Start Emacs

*Observed behaviour:* :eyes: :broken_heart:
![Screenshot from 2020-01-04 15-42-34](https://user-images.githubusercontent.com/44401412/71767339-18ca8b80-2f14-11ea-8b64-42f15b61897b.png)

*Expected behaviour:* :heart: :smile:
![Screenshot from 2020-01-04 15-44-44](https://user-images.githubusercontent.com/44401412/71767361-30a20f80-2f14-11ea-82c6-328f422181fd.png)

#### System Info :computer:
- OS: gnu/linux
- Emacs: 26.3.50
- Spacemacs: 0.300.0
- Spacemacs branch: develop (rev. 9d379e4ff)
- Graphic display: t
- Distribution: spacemacs
- Editing style: hybrid
- Completion: helm
- Layers:
```elisp
(auto-completion
 (better-defaults :variables better-defaults-move-to-beginning-of-code-first t better-defaults-move-to-end-of-code-first t)
 emacs-lisp helm emoji imenu-list search-engine html
 (javascript :variables js2-basic-offset 2 js-indent-level 2)
 markdown multiple-cursors org
 (shell :variables shell-default-height 30 shell-default-position 'bottom shell-default-shell 'eshell)
 treemacs git
 (version-control :variables version-control-diff-tool 'git-gutter+)
 (erc :variables erc-enable-notifications nil erc-server-list
      '(("irc.freenode.net" :port 6697 :ssl t :nick "mphojele")))
 (spell-checking :variables spell-checking-enable-by-default nil)
 gnus helpful copy-as-format
 (dash :variables dash-docs-docset-newpath "~/.local/share/Zeal/Zeal/docsets")
 github pass
 (xclipboard :variables xclipboard-enable-cliphist t))
```
- System configuration features: XPM JPEG TIFF GIF PNG RSVG SOUND GPM DBUS GSETTINGS GLIB NOTIFY ACL GNUTLS LIBXML2 FREETYPE M17N_FLT LIBOTF XFT ZLIB TOOLKIT_SCROLL_BARS GTK3 X11 XDBE XIM MODULES THREADS XWIDGETS LIBSYSTEMD LCMS2

**Comment by duianto** [comment]
There's instructions in the documentation for how to change the separator scale:
https://github.com/syl20bnr/spacemacs/blob/develop/doc/DOCUMENTATION.org#separator-scale

But I can confirm that the separators are shown correctly:
- After starting a server: `emacs --daemon`
- and connecting to it with: `emacsclient -c`

<details><summary>
Pop!_OS 19.10
</summary>
<pre>
#### System Info :computer:
- OS: gnu/linux
- Emacs: 26.3.50
- Spacemacs: 0.300.0
- Spacemacs branch: develop (rev. 96ef7b23a)
- Graphic display: t
- Distribution: spacemacs
- Editing style: vim
- Completion: helm
- Layers:
```elisp
(auto-completion command-log emacs-lisp git helm
                 (javascript :variables javascript-fmt-on-save t)
                 markdown multiple-cursors
                 (org :variables org-agenda-files
                      '("~/org/notes.org"))
                 (shell :variables shell-default-height 30 shell-default-position 'bottom)
                 spell-checking syntax-checking treemacs xclipboard version-control)
```
- System configuration features: XPM JPEG TIFF GIF PNG SOUND DBUS GSETTINGS GLIB NOTIFY LIBSELINUX GNUTLS FREETYPE XFT ZLIB TOOLKIT_SCROLL_BARS GTK3 X11 XDBE XIM MODULES THREADS
</pre>
</details>

**Comment by duianto** [comment]
The author of this issue:
Please change the default font for Spacemacs to get a greet status line #13089

Mentions that the separators look fine when one uses Nerd Fonts:
https://github.com/ryanoasis/nerd-fonts

And when I tested the `DejaVuSansMono Nerd Font` in a PopOS 19.10 virtualbox VM where the separators are slightly shorter than the modeline on startup with the fonts:
`Source Code Pro` and `Fira Mono`

The separators seem to match the modeline height with the nerd font.

**Comment by mphojele** [comment]
I'll check it out, because I changed my font to Iosevka

**Comment by jidibinlin** [comment]
@mphojele You can install the powerline fonts and Nerd fonts to fix it. Both shoud be installed

**Comment by mphojele** [comment]
@jidibinlin any links

**Comment by mphojele** [comment]
I figured that fonts issue was due to Spacemacs deferred loading. Am I wrong? I say that because the modeline is only ugly at startup, but after some commands have run it adjusts accordingly

**Comment by jidibinlin** [comment]
@mphojele You still can install powerline fonts and nerd fonts to fix it.
I have the same problem(ugly spaceline at startup) but when i installed powerline font and nerd font, I got a nice spaceline.

**Comment by jidibinlin** [comment]
@mphojele 
* Both powerline fonts and nerd fonts you can get from github. Just search powerline fonts and nerd fonts in github. 
* Some linux destributions may already have the packages to install powerline or nerd fonts. such archlinux`s yaourt
